internal/eventingester/ingester.go

```diff
@@ -7,6 +7,8 @@ import (
 	"regexp"
 	"sync"
 
+	"github.com/G-Research/armada/internal/lookoutingester/metrics"
```

Comment:
maybe we should move shared metrics to pkg/ instead  of having cross-service dependency here?

Reply:
actually I removed it, as we have better metrics in pulsar itself